.circleci/config.yml

```diff
@@ -24,7 +24,15 @@ jobs:
       GOLANGCI_LINT_VERSION: v1.30.0
     steps:
       - checkout
+      - run:
+          name: fix up GOPATH
+          command: |
+            mkdir -p /go/src/github.com/dominodatalab
+            ln -s $PWD /go/src/github.com/dominodatalab/forge
```

Comment:
Unfortunately, tooling is a different story. Most tooling have not switched or support modules yet.
In this one case, if you run client-gen, it will attempt to use the GOPATH to find and output its generated files. I tried a variety of methods from folks and this was the easiest/cleanest for now.
At some point the tooling will have to support modules.
If you never `generate` then the symlink isn't needed.